src/front-end/typescript/lib/pages/opportunity/code-with-us/lib/index.ts

```diff
@@ -98,6 +103,7 @@ export function cwuOpportunityStatusToPresentTenseVerb(
     case CWUOpportunityStatus.Draft:
       return "Update";
   }
+  return "";
```

Comment:
Same as above, however you need to add a case for CWUOpportunityStatus.UnderReview to render a non-empty string in the toasts.

Reply:
taken care of in https://github.com/bcgov/digital_marketplace/pull/430/commits/d6eff27f9821cb4a669aa36c30115a7b995019a9